login.py

```diff
@@ -45,29 +45,43 @@ def daisy_login(su_username: str, su_password: str, staff: bool = False) -> str:
 
     # 2. Navigate to the login URL which may be needed to retrieve further login form details
     login_response = session.get(
-        "https://daisy.dsv.su.se/Shibboleth.sso/Login?entityID=https://idp.it.su.se/idp/shibboleth&target=https://daisy.dsv.su.se/login_sso_student.jspa" if not staff else "https://daisy.dsv.su.se/Shibboleth.sso/Login?entityID=https://idp.it.su.se/idp/shibboleth&target=https://daisy.dsv.su.se/login_sso_employee.jspa"
```

Comment:
as a general comment please avoid running unwanted formatting tools. It's ok for this repository as the codebase is extremely small